README.md

```diff
@@ -1 +1,128 @@
-# arxiv-search
\ No newline at end of file
+# arxiv-search
+
+## Development quickstart
+
+### Running Elasticsearch + Kibana with docker-compose
+
+The easiest way to spin up ES and Kibana is using the included
+``docker-compose.yml`` file. This will start ES and Kibana on a custom network.
```

Comment:
Also make a note that this can be stopped with `docker-compose down` from the same directory.

Reply:
Thanks; updating

Reply:
### Running Elasticsearch + Kibana with docker-compose

The easiest way to spin up ES and Kibana is using the included
``docker-compose.yml`` file. This will start ES and Kibana on a custom network.
The ES service ports 9200 and 9300, and Kibana service port 5601, will be
mapped to localhost.

```bash
docker-compose up
```
Kibana will be available at http://127.0.0.1:5601/. The containers started by
docker-compose can be stopped with ``docker-compose down`` from the same
directory.

Make sure that you have a recent version of ``docker-compose``; this is
confirmed to work with version 1.18.

Note that connection configuration variables for the search service are set in
``search/config.py``, where they are read from the environment. The arXiv
search service expects the ES service to be available at http://localhost:9200
by default. Hence, you should be able to start ES using docker-compose as above
and make no configuration changes to the arXiv search service.